### ISSUE TITLE
Opengl3 driver shows no GPUParticles2D (CPUParticles2D) when amount is 1

### ISSUE THREAD

**Issue by AldeonMoriak** [godotengine/godot#75951]
### Godot version

4.0.2

### System information

Manjaro, GeForce GT 520M, GLES3

### Issue description

When I create a particle and make its amount 1, nothing gets rendered. It happens for GPUParticles2D and CPUParticles2D. I asked this issue in discord channel and someone confirmed that they had the same issue (in compatibility mode).
For me because I don't have a vulkan supported GPU, even if I run the editor (and game) in forward+ mode still get the same result.

### Steps to reproduce

In compatibility mode:
1. Create a `GPU[CPU]Particles2D` 
2. Change its texture to something (ex: Godot logo)
3. Make its `amount` 1.

The texture should be hidden after step 3
P.S: You could add a `process material` but it wouldn't change anything.

### Minimal reproduction project

N/A

**Comment by clayjohn** [comment]
Duplicate of https://github.com/godotengine/godot/issues/75747

Please remember to search for your issue before opening a new bug report
